src/Cart/Checkout/AddressManagement/AddressesDB.js

```diff
@@ -0,0 +1,22 @@
+export const AddressesDB = [
+  {
+    id: 1,
+    name: "Rishabh",
+    address: "#246, Epitome Colony",
+    city: "Jalandhar",
+    state: "Punjab",
+    country: "India",
+    zip: 152001,
+    mobileNumber: 9877651233
+  },
+  {
+    id: 2,
```

Comment:
Since the mongoDB keeps id as `_id`, it is more flexible to also name the ids as `_id` in the frontend mock database. It will help while syncing backend with frontend.